Running-Mastodon/Production-guide.md

```diff
@@ -188,6 +188,16 @@ and install an ident daemon, which does not come installed by default:
     sudo systemctl start pidentd
     sudo systemctl restart postgresql
 
+### Debian 8
+
+Under Debian 8, the default version of nginx available is too old. To install a newer version of nginx (v1.9.5+), you have to add the jessie-backports repo to your `sources.list`:
```

Comment:
is too old **to work with the proposed nginx config file**. It works perfectly fine if you write your own conf file or if you don't include the http2 thing